1656.py

```diff
@@ -0,0 +1,23 @@
+class OrderedStream:
+
+    def __init__(self, n: int):
+        self.orders = [''] * n
+        self.order_count = 0
+
+    def insert(self, idKey: int, value: str) -> List[str]:
+        self.orders[idKey - 1] = value
+        next_orders = []
+
+        for order in self.orders[self.order_count:]:
+            if not order:
```

Comment:
I haven't solved this problem yet, but I see in many solutions that this `if` block can be avoided. Usually early return statements inside a loop are a result of not-so-well organized logic inside the loop.